### DISCUSSION TITLE
What's wrong with this Viewer URL ?

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [developmentseed/titiler#490] [Q&A]
This works: 
https://titiler.usgs.gov/cog/viewer?url=https://mghp.osn.xsede.org/rsignellbucket1/testing/1998_cog.tif

This doesn't:
https://titiler.usgs.gov/cog/viewer?url=https://mghp.osn.xsede.org/rsignellbucket1/testing/1998_cog.tif&rescale=0%2C2048&bidx=1&bidx=2&bidx=3

What am I doing wrong?   Something simple I'm sure....

**Comment by vincentsarago** [comment]
Hi @rsignell-usgs, you're doing nothing wrong. You're trying to pass query argument to the demo viewer but it only accept `url` (as a shortcut) 

https://github.com/developmentseed/titiler/blob/cce68aa4c79ac1de43bd9fa5e5ebcf8ab81997ae/src/titiler/application/titiler/application/templates/cog_index.html#L825-L832
